### PR TITLE
Add Jupyter definitions

### PR DESCRIPTION
**Pull request by MysteryBlokHed**
## Author: 

- [Adam Thompson-Sharpe](https://github.com/MysteryBlokHed)

## Language: 

- English

## Terms defined:

- Jupyter
- Jupyter Notebook

## Editor

Assign the PR based on the language to the following person:

| Language   | Person                       |
|:----------:|:----------------------------:|
| English    | @baileythegreen, @zkamvar    |

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -3100,6 +3100,28 @@
...
 
+- slug: jupyter
+  ref:
+    - jupyter_notebook
+  en:
+    term: "Jupyter"
+    def: >
+      Project Jupyter is a non-profit open-source project that was born out of the IPython Project
+      in 2014 as it evolved to support interactive [data science](#data_science) and scientific computing
+      in many different programming languages.
```
Comment (baileythegreen) [inline glossary.yml:3111]:
```suggestion
      Project Jupyter is a non-profit, open-source project that was born out of the IPython Project
      in 2014 as IPython evolved to support interactive [data science](#data_science) and scientific
      computing in many different programming languages.
```

File: glossary.yml [hunk 2]
```diff
@@ -3100,6 +3100,28 @@
...
+
+
+- slug: jupyter_notebook
+  ref:
+    - computational_notebook
+    - jupyter
+  en:
+    term: "Jupyter Notebook"
+    def: >
+      The Jupyter Notebook is an open-source web-based [computational notebook](#computational_notebook)
```
Comment (baileythegreen) [inline glossary.yml:3121]:
```suggestion
      The Jupyter Notebook is an open-source, web-based [computational notebook](#computational_notebook)
```

File: glossary.yml [hunk 3]
```diff
@@ -3100,6 +3100,28 @@
...
+      that allows you to write and share live code, equations, visualizations, and narrative text.
```
Comment (baileythegreen) [inline glossary.yml:3122]:
```suggestion
      that allows the user to write and share live code, equations, visualisations, and narrative text.
```
For future reference: British spelling is preferred in The Carpentries.

**Review by baileythegreen — CHANGES_REQUESTED** [review]
These look good. A few small changes. Also, we don't have a definition for IPython, if you would like to add that.

If you haven't already noted this: Jupyter is on the list of terms given in the list in #239. (If that is what prompted this PR, yay! It's working.) When this PR is accepted, it should be noted there.

**Comment by MysteryBlokHed** [comment]
Thank you for the feedback! I've updated the pull request with the suggested changes.

**Review by baileythegreen — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @MysteryBlokHed for translation

**Pull request merged**